<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frms_nv_veiculocliente</class>
 <widget class="QDialog" name="frms_nv_veiculocliente">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>701</width>
    <height>646</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>233</y>
     <width>680</width>
     <height>371</height>
    </rect>
   </property>
   <property name="title">
    <string>Carro Cliente</string>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>240</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Observação</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>47</y>
      <width>511</width>
      <height>52</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_10">
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Marca</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cb_nv_marca"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_11">
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Modelo</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cb_nv_modelo"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_15">
       <item>
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>Ano</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cb_nv_ano"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>178</y>
      <width>401</width>
      <height>52</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_16">
       <item>
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>Placa</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txt_nv_placa"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_13">
       <item>
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Chassi</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txt_nv_chassi"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_3">
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>114</y>
      <width>88</width>
      <height>50</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_12">
     <item>
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Motor</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cb_nv_motor"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_4">
    <property name="geometry">
     <rect>
      <x>116</x>
      <y>114</y>
      <width>144</width>
      <height>50</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_14">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Cor</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txt_nv_cor"/>
     </item>
    </layout>
   </widget>
   <widget class="QPlainTextEdit" name="txt2_nv_observacao">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>260</y>
      <width>201</width>
      <height>91</height>
     </rect>
    </property>
    <property name="plainText">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QTableWidget" name="tw_nv_listaveiculosclientes">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>55</y>
     <width>681</width>
     <height>171</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_nv_filtrarcliente">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>20</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Filtrar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_salvarveiculo">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>613</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Salvar</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>19</y>
     <width>192</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Filtrar</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txt_nv_filtrarcliente"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
